bucket/imagemagick.json

```diff
@@ -31,17 +34,26 @@
         "montage.exe",
         "stream.exe"
     ],
+    "env_set": {
+        "MAGICK_HOME": "$dir",
+        "MAGICK_CODER_MODULE_PATH": "$dir\\modules\\coders"
+    },
+    "env_add_path": ".",
+    "post_install": [
+        "Rename-Item $dir\\ffmpeg.exe $dir\\ffmpeg.exe.old"
+    ],
+    "notes": "The bundled ffmpeg is outdated and has been renamed to ffmpeg.exe.old - if you need it, run 'scoop install ffmpeg' for the latest version",
```

Comment:
Now that you have removed the dependency, you can remove the post_install and note.

Reply:
However, if I remove the post_install, ffmpeg will be added to the path (because of the `"env_add_path": "."`) Then if the user  decides later to install ffmpeg they will have two conflicting ffmpegs in their path. I think it should be kept for that reason.

Reply:
The thing is, they have probably kept an older version of ffmpeg to prevent any incompatibilities with a newer version. Suggesting users to install the latest version via `scoop install ffmpeg` might cause issues with ImageMagick.

Also, I just noticed another thing - we are creating shims for binaries that are already in path (because of env_add_path) - which is very wasteful...

Reply:
Yes it's possible it'll cause issues. So if I leave ffmpeg in the path and the user then installs ffmpeg from scoop, there will be a conflict. And they will have to manually rename/remove the imagemagick version every time there is a new version of imagemagick... Is there a better way?

I didn't notice that, nice catch. To resolve that I can remove all of the shims. Apart from convert.exe which I will have to rename to imconvert.exe in post_install.

Reply:
Let's just not suggest users to install the latest version. We can change the note to say - "The bundled ffmpeg has been renamed to 'ffmpeg.exe.old' to prevent conflict with a standalone ffmpeg installation - if you need it for ImageMagick tools, rename it back to 'ffmpeg.exe'."

Reply:
Okay, done.